packages/lesswrong/components/editor/Editor.tsx

```diff
@@ -132,12 +132,11 @@ export const editorTypeToDisplay: Record<EditorTypeString,{name: string, postfix
   draftJS: {name: 'Draft-JS'},
 }
 
-export const nonAdminEditors: EditorTypeString[] = ['ckEditorMarkup', 'markdown', 'draftJS']
+export const nonAdminEditors: EditorTypeString[] = ['ckEditorMarkup', 'markdown']
```

Comment:
What happens to the editor-select widget if you open a document that's already in draftjs?

Reply:
It shows the warning message at the top for "you are using draftJS instead of your default editor, [insert default editor, probably LessWrong docs]. Click [here] to convert to LessWrong docs."

You can continue editing old draftJS docs. Converting them is a one-way-trip (although if something goes wrong and you really need to revert you should be able to ping an admin)